src/main/java/dev/ionfusion/fusion/_private/doc/model/BindingDoc.java

```diff
@@ -60,29 +59,11 @@ public void setKind(Kind kind)
 
     public String getUsage()
     {
-        if (myUsage != null
-            && ! (myUsage.startsWith("(") && myUsage.endsWith(")")))
-        {
-            assert false: "I think this is unused.";
-            StringBuilder buf = new StringBuilder();
-            buf.append('(');
-            buf.append(myName == null ? "_" : myName);
-            if (! myUsage.isEmpty())
-            {
-                if (! myUsage.startsWith(" ")) buf.append(' ');
-                buf.append(myUsage);
-            }
-            buf.append(')');
-
-            myUsage = buf.toString();
-        }
-
         return myUsage;
     }
 
     void setUsage(String usage)
     {
-        assert false: "I don't think this can happen.";
```

Comment:
This method is unused: The `(define (proc args...)` form will synthesize a usage string and prepend it to the body docs. ([In Fusion code, no less!](https://github.com/ion-fusion/fusion-java/blob/3b5915c2cef3a3ccc7a493cb4ad94e6861b4e160/fusion/src/fusion/private/define.fusion#L99-L117))

That said, I want to keep this for now since the above should be handled distinctly.